services/services.go

```diff
@@ -113,12 +113,12 @@ func WithConfig(c toml.Primitive) ServicerFunc {
 	}
 }
 
-func WithScripter(scr scripter.Scripter) ServicerFunc {
```

Comment:
Missing comments

Reply:
Proven that comments do not matter in this file. Will be requested in new issue.